container/go/cmd/pusher/pusher.go

```diff
@@ -126,7 +127,12 @@ func main() {
 		log.Printf("Failed to digest image: %v", err)
 	}
 
-	if err := push(stamped, img); err != nil {
+	var options = []name.Option{};
```

Comment:
This needs to be run through gofmt, Go code does not typically have semi colons.

Also, it's usually preferred to write this as:

```go
var opts []name.Option
```